NitroxPatcher/Patches/Dynamic/BaseGhost_Place_Patch.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Reflection;
+using Harmony;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.Logger;
+using static NitroxClient.GameLogic.Helper.TransientLocalObjectManager;
+
+namespace NitroxPatcher.Patches.Dynamic
+{
+    public class BaseGhost_Place_Patch : NitroxPatch, IDynamicPatch
```

Comment:
It looks like you are trying to merge deleted files.  Can you verify that you were originally testing on the tip of master?

Reply:
Does it need to be removed or replaced or slightly changed, I did not add this, this was there when I got my hands on this code

Reply:
i didn't even edit this file